src/getStore.spec.lua

```diff
@@ -6,57 +6,32 @@ return function()
 
 	local getStore = require(script.Parent.getStore)
```

Comment:
We may actually need to deprecate getStore altogether! I'm not sure there's a way to do it with this API, and I think the only use cases for it can actually be cleaned up. We should take a closer look at them.